Task6.2.2

```diff
@@ -0,0 +1,22 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+def is_positive(number):
+    """
+    This function returns true if the number is positive else returns false.
+    :param: number.
+    :return:returns true if the number is positive else returns false.
```

Comment:
This is not what you were asked to do in the assignment, please got over it and resubmit.